SPECS/kernel/kernel.spec

```diff
@@ -302,9 +302,9 @@ make INSTALL_MOD_PATH=%{buildroot} modules_install
 # Verify for build-id match
 # We observe different IDs sometimes
 # TODO: debug it
-ID1=`readelf -n vmlinux | grep "Build ID"`
+ID1=`eu-readelf -n vmlinux | grep "Build ID"`
```

Comment:
Not sure why we're doing this change, but shouldn't we do the same thing inside `kernel-hyperv.spec`? Could you mention this change inside the changelog as well?